shared/video-sdk/develop/spatial-audio/project-test/unity.mdx

```diff
@@ -11,10 +11,10 @@
 </ProductWrapper>
 
 <ProductWrapper notAllowed="interactive-live-streaming">
-5. Press **Join** to connect to the same channel as your web demo.
+6. Press **Join** to connect to the same channel as your web demo.
 </ProductWrapper>
 
-6. **Test spatial audio effects for remote users**
+7. **Test spatial audio effects for remote users**
```

Comment:
Test spatial audio effects for media player is no 7 as well. I've fixed it, but please take a bit more care.